server/api/like.go

```diff
@@ -0,0 +1,107 @@
+package handler
+
+import (
+	"net/http"
+
+	"github.com/codeday-labs/bookworms/server/db"
+	"github.com/codeday-labs/bookworms/server/utils"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+type newLikeBody struct {
+	ReviewID string `json:"review_id"`
+}
+
+func likeReview(id string, review *db.Review) error {
+
+	DB, err := db.DB()
+
+	if err != nil {
+		return err
+	}
+
+	objectID, err := primitive.ObjectIDFromHex(id)
+
+	if err != nil {
+		return err
+	}
+
+	_, err = DB.Collection(db.ReviewsCollection).UpdateOne(
+		db.Ctx,
+		bson.M{"_id": objectID},
+		bson.D{
+			{"$inc", bson.D{{"likes", 1}}},
+		},
+		options.Update().SetUpsert(true),
+	)
+
+	if err != nil {
+		return err
+	}
+
+	review.Likes += 1
+
+	defer DB.Client().Disconnect(db.Ctx)
+
+	return nil
+}
+
+func FindReview(id string, review *db.Review) error {
```

Comment:
function with first letter capitalized means it is exported function and ready to be shared with other package, if it is not your intention here, shoud use lower case here.